staging/src/k8s.io/apiserver/pkg/endpoints/filterlatency/filterlatency.go

```diff
@@ -76,21 +76,20 @@ func trackStarted(handler http.Handler, tp trace.TracerProvider, name string, cl
 	// This is a noop if the tracing is disabled, since tp will be a NoopTracerProvider
 	tracer := tp.Tracer("k8s.op/apiserver/pkg/endpoints/filterlatency")
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		ctx := r.Context()
+		ctx, _ := tracer.Start(r.Context(), name)
+		defer trace.SpanFromContext(ctx).End()
```

Comment:
This can be simplified.
```suggestion
		ctx, span := tracer.Start(r.Context(), name)
		defer span.End()
```

Reply:
Good suggestion